microsoft-edge/devtools-guide-chromium/whats-new/2023/02/devtools-110.md

```diff
@@ -0,0 +1,168 @@
+---
+title: What's new in DevTools (Microsoft Edge 110)
+description: "Faster heap snapshots in Memory tool. Move tools between Activity Bar and Quick View in Focus Mode. Screen readers report when More Tools is expanded. Change Dock and Activity Bar location via keyboard. Screen readers announce which theme is selected. Improved support for contrast themes. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/09/2023
+---
+# What's New in DevTools (Microsoft Edge 110)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Taking heap snapshots in the Memory tool is now faster
+
+<!-- Subtitle: When taking a heap snapshot greater than 1 GB in size, the time spent generating the snapshot in the Memory tool is at least 70% faster. -->
+
+In previous versions of Microsoft Edge, generating large heap snapshots (over 1 GB) in the **Memory** tool could take anywhere from 2 to 15 minutes, depending on the size of your website or app, and how many elements were stored in memory.
+
+In Microsoft Edge 110, the **Memory** tool is now 70% to 86% faster at taking heap snapshots.  For larger heap snapshots, we're seeing the greatest improvement in speed.  If you've noticed that taking heap snapshots is much faster, please leave us feedback in [[Feedback] Taking heap snapshots in the Memory tool is now faster #131](https://github.com/MicrosoftEdge/DevTools/issues/131).
+
+![Taking a heap snapshot in the Memory tool](./devtools-110-images/faster-heap-snapshot.png)
+
+See also:
+* [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md#take-a-snapshot)
+
+
+<!-- ====================================================================== -->
+## Move tools between the Activity Bar and Quick View more easily in Focus Mode
+
+<!-- Subtitle: Customize both the Activity Bar and Quick View in Focus Mode to see only the tools you care about. -->
+
+Microsoft Edge 110 contains various improvements to **Focus Mode** in DevTools.  The following changes make it easier to customize the Activity Bar and Quick View, to show only the tools that you're currently interested in.
```

Comment:
I know we have a "see also" link at the bottom of this section. Should we also include a link here? Otherwise, one would have to read the whole thing before finding the link.

https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode?branch=pr-en-us-2427

Reply:
Microsoft Edge 110 contains various improvements to [Focus Mode](../../../experimental-features/focus-mode.md) in DevTools.